.github/CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# This file controls default reviewers for LightGBM code.
+# See https://help.github.com/en/articles/about-code-owners
+# for details
+#
+# Maintainers are encouraged to use their best discretion in
+# setting reviewers on PRs manually, but this file should
+# offer a reasonable automatic best-guess
+
+# main C++ code
+include/*    @guolinke @chivee
+CmakeLists.txt    @guolinke @chivee @Laurae2 @jameslamb @wxchan @henry0312 @StrikerRUS @huanzhang12
+
+# R code
+R-package/*    @Laurae2 @jameslamb
+*.R    @Laurae2 @jameslamb
+
+# Python code
+python-package/*    @StrikerRUS @chivee
+*.py    @StrikerRUS @wxchan @henry0312
```

Comment:
This regex catches files in https://github.com/microsoft/LightGBM/tree/master/helpers folder, for example. Not sure that anyone else beside me and **guolinke**  want to maintain them...

Reply:
To be honest, I never even noticed that `helpers/` directory was there!

I'll remove it and rely on `python-package/*` to catch things